### PR TITLE
*: Fix makefile build cmd

### PR DESCRIPTION
**Pull request by Prnyself**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
+	@echo "  build               to create bin directory and build beyond-ftp"
 	@echo "  generate            to generate code"
 	@echo "  test                to run test"
 
-check: vet
+check: format vet
 
 format:
-	go fmt ./...
+	@echo "go fmt"
```
Comment (Xuanwo) [inline Makefile]: How about remove version echo here?

File: Makefile [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
+	@go fmt ./...
+	@echo "ok"
```
Comment (Xuanwo) [inline Makefile]: ditto

File: Makefile [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
 
 generate:
 	@echo "generate code"
 	go generate ./...
-	go fmt ./...
+	@echo "ok"
 
 build: tidy generate check
-	go build ./...
+	@echo "build beyond-ftp"
```
Comment (Xuanwo) [inline Makefile:25]: ditto.

File: Makefile [hunk 4]
```diff
@@ -33,8 +45,10 @@ integration_test:
 	go test -race -count=1 -covermode=atomic -v ./tests
 
 tidy:
-	go mod tidy
-	go mod verify
+	@echo "Tidy and check the go mod files"
```
Comment (Xuanwo) [inline Makefile]: ditto

File: Makefile [hunk 5]
```diff
@@ -1,29 +1,33 @@
...
-
-vet:
-	go vet ./...
+	@echo "ok"
 
 generate:
-	@echo "generate code"
 	go generate ./...
-	go fmt ./...
+	@echo "ok"
```
Comment (Xuanwo) [inline Makefile]: Do we really need an `ok` or `done` here?

Comment (Prnyself) [reply Makefile]: From my perspective, an `ok` here makes me feel secure to know it works well.

But I can accept to remove them :)

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**